### ISSUE TITLE
LM4040 - wrong pinout?

### ISSUE THREAD

**Issue by ultrapalace** [barnabywalters/Schraeg#2]
The two LM4040s have footprints with anode at pin1 and cathode at pin 3. In the SOT23 package, the actual anode is on pin 2, with the cathode on pin 1, and pin 3 must be connected to anode or left floating. Rotating my LM4040's clockwise allows generation of the proper 5V and -5V references.

I have not yet found an LM4040 with a footprint as is drawn in the circuit, but if there's one that works, please update the BOM to specify that one.

![2021-05-28 10_43_47-LM4040-N_-Q1 Precision Micropower Shunt Voltage Reference datasheet (Rev  K)](https://user-images.githubusercontent.com/81344042/120022689-9f688880-bfa1-11eb-8e41-393c10fa245d.png)
![2021-05-28 10_43_22-Pcbnew — C__Users_cctv_Downloads_Schraeg-master_Schraeg_Schraeg kicad_pcb](https://user-images.githubusercontent.com/81344042/120022697-a1324c00-bfa1-11eb-8cec-7fad617212f6.png)

**Comment by barnabywalters** [comment]
Well spotted! I think this was caused by the switch from the old LM4040 package and the new one. I made the mistake of assuming that KiCAD would correctly map the old schematic symbols to the new footprints, but it didn’t, and I had to swap out the schematic symbol too.

It looks like this may have been working already in some cases as “Pin 3 is attached to substrate and must be connected to ANODE or left open.“, which would explain why the module worked well enough already, and I didn’t notice any problems.
